people_recognition_3d/src/people_recognition_3d/people_recognizer_3d.py

```diff
@@ -496,6 +515,16 @@ def get_person_tags(self, skeleton):
                 elif wrist.x > (other.x + self._hor_threshold):
                     tags.append(side + 'Pointing')
 
+                try:
+                    elbow = skeleton[side + 'Elbow'].point
+                except KeyError:
+                    pass
+                else:
+                    if other.y < wrist.y < elbow.y:
```

Comment:
`other` comes from 'Shoulder' so maybe rename `other` to `shoulder`. This would also clear up the logic further

Reply:
Made the change. However when I look into the logic after this change, I see flaws in the if conditions along x-direction. They work because of heuristics, but we can obviously do better.